CONTRIBUTING.md

```diff
@@ -12,6 +12,45 @@ npm install
 
 Make changes on a feature branch, then open a pull request. Make sure CI passes on your branch, and you include any relevant new tests.
 
+### Generating measures-data.json
+To regenerate the `measures-data.json` and `measures-data.xml` files, which contain additional metadata and conform to
+the measures schema, do the following:
+
+```
+npm start
```

Comment:
Can we use a different verb here? start doesn't feel correct since the default behavior is to start a server.